docs/content/how-to/azure/create-azure-cluster-with-options.md

```
@@ -1,269 +1,278 @@
+      name: azure-20514bc7
+      timeout: 35s
+  - identity: {}
+  resources:
+  - secrets
+  - configmaps
+  - routes.route.openshift.io
+  - oauthaccesstokens.oauth.openshift.io
+  - oauthauthorizetokens.oauth.openshift.io
+```
+
+Next, confirm the ` azure-kms-provider-active` container in the kube-apiserver pod is running properly, there are no 
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)